### PR TITLE
CANS-964 - Remove/Add Caregiver links

### PR DESCRIPTION
**Pull request by abelayt**
## Tests

- [ ] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/style.sass [hunk 1]
```diff
@@ -196,20 +196,17 @@ $item-confidential-label-height: 6.05rem
...
-  color: $aspen
-  cursor: pointer
-  font-weight: bold
+.caregiver-domain-add-remove
+      float: right
+      margin-top: 2.0rem
+      margin-right: 1.5rem
+
+.caregiver-domain-add-remove-links
+      font-size: 1.4rem
```
Comment (denys-davydov) [inline app/javascript/Application/components/Assessment/style.sass]: "ADD CANS" button on the Client Profile page has a size of 12.5px. AC #4 states that "The text links are designed to match the font size of the "Add CANS" text link on the client profile"

File: app/javascript/Application/components/Assessment/DomainCaregiverControls.js [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
-              aria-label="add caregiver button"
-              tabIndex={0}
-            >
-              + ADD CAREGIVER
-            </div>
-          </li>
-        </ul>
-      </h5>
-    </div>
+    <Fragment>
```
Comment (denys-davydov) [inline app/javascript/Application/components/Assessment/DomainCaregiverControls.js]: There is no need in Fragment if it has only one child, like here it has only a div as a child

File: app/javascript/Application/components/Assessment/style.sass [hunk 3]
```diff
@@ -196,20 +196,17 @@ $item-confidential-label-height: 6.05rem
...
-  font-weight: bold
+.caregiver-domain-add-remove
+      float: right
+      margin-top: 2.0rem
+      margin-right: 1.5rem
+
+.caregiver-domain-add-remove-links
+      font-size: 1.25rem
+      font-weight: bold;
+      color: $light-blue
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/style.sass]: Are these styles necessary? When using `<Button color="link"/>` from the component library, I would expect the component library to provide styles for font-weight, color, and text-decoration.

If we are overriding component library styles for this particular situation, we should consider checking with DesignOps to see if that is really necessary. The component library should be as close to a drop-in solution as possible.

Comment (abelayt) [reply app/javascript/Application/components/Assessment/style.sass]: I checked with Robert from DesignOps and showed him the links before I changed the font-sizes as Denys mentioned from the A/C (from 1.4rem to 1.25rem). Robert approved and said they are still working on the global design for CWS-CARES  and these and many others will be changed. Your point, we should not override component library styles, should still be the case even now with what we have in the component library, but that is not the case with some of the styles as of now.

Comment (wmitchell-cambria) [reply app/javascript/Application/components/Assessment/style.sass]: If these styles will change, it seems even more important to use the default component library styles. That way, when the global styles are updated, CANS will not lag behind.

Comment (abelayt) [reply app/javascript/Application/components/Assessment/style.sass]: The default component library styles does not meet the A/C for this story. That is the reason we need to style to match what we have globally.

**Comment by abelayt** [comment]
retest this please

**Pull request merged**